lib/matplotlib/tests/test_axes.py

```diff
@@ -9720,3 +9720,11 @@ def test_bar_shape_mismatch():
     )
     with pytest.raises(ValueError, match=error_message):
         plt.bar(x, height)
+
+
+def test_pie_non_finite_values():
+    fig, ax = plt.subplots()
+    df = [5, float('nan'), float('inf')]
```

Comment:
```suggestion
    values = [5, float('nan'), float('inf')]
```
Optional. `df` is typically used as variable name for dataframes, so it's slightly misleading here.